cloudfront/wellcomelibrary.org/edge-lambda/testRedirects.ts

```diff
@@ -56,75 +77,88 @@ async function testRedirects(tests: RedirectTest) {
     })
   );
 
-  return {
-    host: tests.host,
+  redirectTestSet.results = {
+    host: host,
     results: testResults,
-  } as ResultSet;
+  };
+
+  return redirectTestSet;
 }
 
-// Blog tests
-const getBlogTests: RedirectTestSet = (env: EnvId) => {
-  const blogEnvs: HostEnvs = {
-    stage: 'http://blog.stage.wellcomelibrary.org/',
-    prod: 'http://blog.wellcomelibrary.org/',
-  };
+const displayResultSet = (redirectTestSet: RedirectTestSet) => {
+  console.log(chalk.blue.underline.bold(`\n${redirectTestSet.displayName}`));
 
-  const host = env === 'stage' ? blogEnvs.stage : blogEnvs.prod;
+  if (redirectTestSet.results) {
+    const resultSet = redirectTestSet.results;
+    console.log(chalk.blue.bold(`${resultSet.host}`));
 
-  const blogRedirectPaths = {
-    '/2018/05/goodbye-from-wellcome-library-blog/':
-      'https://wayback.archive-it.org/16107-test/20210302041159/http://blog.wellcomelibrary.org/2018/05/goodbye-from-wellcome-library-blog/',
-    '/':
-      'https://wayback.archive-it.org/16107-test/20210301155649/http://blog.wellcomelibrary.org/',
-  };
+    resultSet.results.forEach((result) => {
+      if (result.error) {
+        console.error(chalk.red(`✘ ${result.from}\n${result.error}`));
+      } else {
+        console.info(chalk.green(`✓ ${result.fromPath}`));
+      }
+    });
+  } else {
+    console.info('No results');
+  }
 
-  return {
-    host: host,
-    pathTests: blogRedirectPaths,
-  };
+  return redirectTestSet;
+};
+
+const itemTestSet = {
+  displayName: 'Item pages',
+  fileLocation: 'itemRedirects.csv',
+  fileHostPrefix: 'wellcomelibrary.org',
+  headers: staticRedirectHeaders,
+  envs: {
+    stage: 'http://stage.wellcomelibrary.org',
+    // prod: 'http://wellcomelibrary.org',
+  },
+};
+
+const blogTestSet = {
+  displayName: 'Library blog',
+  fileLocation: 'blogRedirects.csv',
+  fileHostPrefix: 'blog.wellcomelibrary.org',
+  headers: ['sourceUrl', 'targetUrl'],
+  envs: {
+    stage: 'http://blog.stage.wellcomelibrary.org/',
+    // prod: 'http://blog.wellcomelibrary.org/',
```

Comment:
?

Reply:
It is not yet available in the prod environment, (actually will be though later today).